build/_build.ts

```diff
@@ -31,9 +31,12 @@ async function main() {
       .execSync("func -V")
```

Comment:
I think we wanted to use blueprint, which should make all scripts under `build/` obsolete

Reply:
Right. old build scripts use system binaries for compiling contracts. So we should switch to func-js? which version?

Reply:
Yes, we can use the version blueprint comes with